backend/src/main/cards/cards.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC
+
+
+class Cards(ABC):
```

Comment:
Rename to something like `CardCollection` or `CardDeck` to be more obvious than `Card` vs `Cards`